README.md

```diff
@@ -56,6 +56,8 @@ iex> AMQP.Basic.publish chan, "test_exchange", "", "Hello, World!"
 Received: Hello, World!
 ```
 
+To avoid problems with terminate in 60 seconds as you opened the connection, pass `heartbet: 30` to `AMQP.Connection.opne()` like argument.
```

Comment:
There is a typo in the `open` function name.

Moreoever, the problem definition is very vague. It sounds like a [proxy effect](http://www.rabbitmq.com/heartbeats.html#tcp-proxies) which is environment-specific.

Please reword this to be a generic heartbeat example and link to the [heartbeat guide](http://www.rabbitmq.com/heartbeats.html#tcp-proxies) on rabbitmq.com.

Reply:
totally agree! thanks :+1: